web/auth/jwt.go

```diff
@@ -56,39 +55,35 @@ func (tm *TokenManager) JWTUpdateClaims(db database.Database, userID uint64) (*C
 	return newClaims, nil
 }
 
+// TODO(vera): needs access to the database to update changes there
 // Update will remove a user ID at the index from the list
-func (tm *TokenManager) Remove(index uint64) {
+func (tm *TokenManager) Remove(index uint64, db database.Database) {
 	updatedTokenList := append(tm.TokensToUpdate[:index], tm.TokensToUpdate[index+1:]...)
 	tm.TokensToUpdate = updatedTokenList
 }
 
 // Add adds a new UserID to the list of users who require an updated JWT token
-func (tm *TokenManager) Add(userID uint64) {
+func (tm *TokenManager) Add(userID uint64, logger *zap.SugaredLogger, db database.Database) {
 	// Return if the given ID is already in the list to avoid duplicates
-	for _, id := range tm.TokensToUpdate {
-		if userID == id {
+	for _, token := range tm.TokensToUpdate {
+		if userID == token.UserID {
 			return
 		}
 	}
-	tm.TokensToUpdate = append(tm.TokensToUpdate, userID)
-}
-
-// String creates a string of all IDs in the list to store in the database
-func (tm *TokenManager) String() string {
-	return strings.Trim(strings.Join(strings.Split(fmt.Sprint(tm.TokensToUpdate), " "), ","), "[]")
+	tokenQuery := &pb.UpdateTokenRecord{UserID: userID}
+	if err := db.CreateTokenRecord(tokenQuery); err != nil {
+		logger.Errorf("error updating token record in the database: %w", err)
```

Comment:
When logging errors in methods/functions we must do:
- either return after logging the error or
- add a comment explaining why we don't return
- another option is always to return the `err`

Also:
- The capitalization rule does not apply when logging. 
- The message could be: `Failed to update JWT token in database: %v`
- Recall that `%w` does not apply to printing/logging, only if you use `fmt.Errorf("... : %w)` to construct a new error with added context that you return to the caller.